### PR TITLE
Fix file upload to support non-ASCII file names

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** allow non-ASCII file names in ORA uploads.

JIRA: [EDUCATOR-5566](https://openedx.atlassian.net/browse/EDUCATOR-5566)

**What changed?**

- Modify upload headers to encode the filename, allowing for non-ASCII file names

**Developer Checklist**

- [x] Reviewed the [release process](./release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

**Testing Instructions**

1. Open an Open Response Assessment with file uploads enabled
2. Attempt to upload a file with non-ASCII characters in the name (e.g. `tęšt-fïłê.png`) 
3. Upload should succeed (previously, uploading file would fail with an error on the frontend)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/xblock/static/js/spec/lms/oa_file_upload.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ describe("OpenAssessment.FileUploader", function() {
             async: false,
             processData: false,
             contentType: 'image/jpeg',
-            headers: { 'Content-Disposition': `attachment; filename="${TEST_FILE.name}"` },
+            headers: { 'Content-Disposition': `attachment; filename*=UTF-8''${encodeURI(TEST_FILE.name)}` },
```
Comment (jansenk) [inline openassessment/xblock/static/js/spec/lms/oa_file_upload.js:39]: What's with the two single quotes between `UTF-8` and `${` ?

**Review by jnlapierre — APPROVED** [review]
Also would like to know what the two quotes means, but otherwise LGTM

File: openassessment/xblock/static/js/spec/lms/oa_file_upload.js (hunk 1 above)
Comment (nsprenkle) [reply openassessment/xblock/static/js/spec/lms/oa_file_upload.js:39]: ^ It optionally includes language info (e.g. `utf-8'en'<filename>`) but can be omitted if we don't have that language meta

Comment (jansenk) [reply openassessment/xblock/static/js/spec/lms/oa_file_upload.js:39]: I don't totally understand, but I understand enough to say this looks good to me

**Review by jansenk — APPROVED** [review]
